practical/jython/open_image.jy

```diff
@@ -18,7 +18,7 @@
 # ------------------------------------------------------------------------------
 
 # This script downloads a file and opens it in ImageJ using Bio-Formats
-# exporter.
+# importer.
```

Comment:
Aha, it is fixed here in jython. Unfortunately, the ``exporter`` made it already to the groovy script in the [groovy PR](https://github.com/ome/training-scripts/pull/28#issuecomment-412039417) in the meantime, so I was confused about it [in that PR](https://github.com/ome/training-scripts/pull/28#issuecomment-412039417)